spec/models/task_spec.rb

```diff
@@ -270,3 +270,47 @@
     end
   end
 end
+  describe "#mark_completed!" do
+    let(:current_time) { Time.parse("2025-01-15 12:00:00") }
+
+    before { travel_to(current_time) }
+    after { travel_back }
```

Comment:
I'd prefer using an `around` block here, rather than a "before that needs an after"